packages/pegasus/src/pegasus.js

```diff
@@ -273,287 +54,348 @@ const makePegasus = (zcf, board, namesByAddress) => {
+        // additional UnhandledRejectionWarning in our vat.
+        promise.catch(() => {});
+        reject(assert.error(X`${remoteDenom} is temporarily unavailable`));
+
+        // Allow new transfers to be initiated.
+        remoteDenomToCourierPK.delete(remoteDenom);
+      },
+      async pegRemote(
+        allegedName,
+        remoteDenom,
+        assetKind = undefined,
+        displayInfo = undefined,
+      ) {
+        checkAbort();
+        const { remoteDenomToCourierPK } = localDenomState;
+
+        // Create the issuer for the local erights corresponding to the remote values.
+        const localKeyword = createLocalIssuerKeyword();
+        const zcfMint = await zcf.makeZCFMint(
+          localKeyword,
+          assetKind,
+          displayInfo,
+        );
+        checkAbort();
+        const { brand: localBrand } = zcfMint.getIssuerRecord();
+
+        // Describe how to retain/redeem pegged shadow erights.
+        const courier = makeCourier({
+          zcf,
+          localBrand,
+          board,
+          namesByAddress,
+          remoteDenom,
+          retain: (zcfSeat, amounts) =>
+            zcfMint.burnLosses(harden(amounts), zcfSeat),
+          redeem: (zcfSeat, amounts) => {
+            zcfMint.mintGains(harden(amounts), zcfSeat);
+          },
+          transferProtocol,
+        });
+
+        const courierPK = getCourierPK(remoteDenom, remoteDenomToCourierPK);
+        courierPK.resolve(courier);
+
+        checkAbort();
+        const peg = makePeg(localDenomState, {
+          localBrand,
+          remoteDenom,
+          allegedName,
+        });
+        pegs.add(peg);
+        return peg;
+      },
+
+      async pegLocal(allegedName, localIssuer) {
+        checkAbort();
+
+        // We need the last nonce for our denom name.
+        localDenomState.lastDenomNonce += 1;
+        const remoteDenom = `pegasus${localDenomState.lastDenomNonce}`;
+
+        // Create a seat in which to keep our denomination.
+        const { zcfSeat: poolSeat } = zcf.makeEmptySeatKit();
+
+        // Ensure the issuer can be used in Zoe offers.
+        const localKeyword = createLocalIssuerKeyword();
+        const { brand: localBrand } = await zcf.saveIssuer(
+          localIssuer,
+          localKeyword,
+        );
+        checkAbort();
+
+        /**
+         * Transfer amount (of localBrand) from loser to winner seats.
+         *
+         * @param {Amount} amount amount to transfer
+         * @param {Keyword} loserKeyword the keyword to take from the loser
+         * @param {ZCFSeat} loser seat to transfer from
+         * @param {Keyword} winnerKeyword the keyword to give to the winner
+         * @param {ZCFSeat} winner seat to transfer to
+         */
+        const transferAmountFrom = (
+          amount,
+          loserKeyword,
+          loser,
+          winnerKeyword,
+          winner,
+        ) => {
+          // Transfer the amount to our backing seat.
```

Comment:
our backing seat... that's `poolSeat`, right? I don't see any reference to it in this function. Is `poolSeat` is always the winner? If so, why pass it in?

Reply:
I think I see what's going on now...
```suggestion
          // Transfer the amount to / from our backing seat.
```